docs/01-start-here/05-development-tips.md

```diff
@@ -2,6 +2,11 @@
 
 ## Client-side development
 
+## Clean all projects
```

Comment:
Maybe under this heading instead: https://github.com/guardian/frontend/blob/78985d0cef88c1d810512241fc5243821ce49177/docs/01-start-here/05-development-tips.md#server-side-development